iModelCore/iModelPlatform/PublicAPI/DgnPlatform/Render.h

```diff
@@ -621,8 +621,8 @@ struct TextureMapping
     struct Trans2x3
     {
         double m_val[2][3];
-        Trans2x3() {}
-        Trans2x3(double t00, double t01, double t02, double t10, double t11, double t12)
+        DGNPLATFORM_EXPORT Trans2x3() {}

```

Comment:
Did you find these necessary? It won't hurt anything, but functions defined inline shouldn't need an export attribute.

Reply:
Yes, you're right.  I misread the error & thought the constructor was what it was complaining about, but it was the FromJason.  I'll remove it from the constructors & test it out to make sure.

Reply:
OK, Done.